apps/et/et-sya/ithc-image-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
```

Comment:
need to add the image policy into automation/kustomization.yaml as well

Reply:
Thanks trying to do this whilst in meetings and being pinged constantly. Will sort @hsjhita.